radp/digital_twin/tests/rf/bayesian/test_bayesian_engine.py

```diff
@@ -200,7 +203,94 @@ def split_training_and_test_data(ue_data_df: pd.DataFrame, test_size: float):
     return cell_id_training_data_map, cell_id_test_data_map
 
 
+def seed_everything(seed: int):
+    random.seed(seed)
+    os.environ["PYTHONHASHSEED"] = str(seed)
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed(seed)
+    torch.backends.cudnn.deterministic = True
+    torch.backends.cudnn.benchmark = True
+
+
 class TestBayesianDigitalTwin(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        seed_everything(1)
```

Comment:
only this needs to be here -- the bottom values need not be class values, since you are only using it in one method. Feel free to move this there.